links.json

```diff
@@ -29,7 +29,7 @@
                     "format": "uri-template"
                 },
                 "hrefSchema": {
-                    "$ref": "http://json-schema.org/draft-08/hyper-schema#"
+                    "$recursiveRef": "http://json-schema.org/draft-08/hyper-schema"
```

Comment:
When using links.json as a standalone schema, it is the entry point schema, and for the initial evaluation it does not matter whether this is a `"$ref"` or a `"$recursiveRef"`.  This schema does _not_ set `"$recursiveRoot"`, so the literal URI reference value provided here is used as-is.

When links.json is referenced from hyper-schema.json (with hyper-schema.json as the entry point), then we have seen as `"$recursiveRoot"`, so the literal value is ignored and the target is the root schema of hyper-schema.json

When links.json is referenced from hyper-schema.json, when the entry point was hyper-operations.json, then the `"$recusiveRoot"` in hyper-operations.json is the first encountered, so the target of this (and other) `"$recursiveRef"`s is the root schema of hyper-operations.json

These behaviors show how the same links.json can be used on its own (referring to hyper-schema.json which extends schema.json) and as a component within hyper-schema.json (or an extension of it, such as hyper-operations.json).